Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.h

```diff
@@ -391,7 +391,7 @@ class UnifiedPDFPlugin final : public PDFPluginBase, public WebCore::GraphicsLay
     void extendCurrentSelectionIfNeeded();
     void updateCurrentSelectionForContextMenuEventIfNeeded();
     void continueTrackingSelection(PDFDocumentLayout::PageIndex, const WebCore::FloatPoint& pagePoint, IsDraggingSelection);
-    void freezeCursorDuringSelectionDragIfNeeded(IsDraggingSelection);
+    void freezeCursorDuringSelectionDragIfNeeded(IsDraggingSelection, bool isMarqueeSelection);
```

Comment:
Might be nice to fold both into a single `OptionSet` parameter, or make this parameter an enum class for consistency with the other one.

(don't feel strongly though)

Reply:
Yeah I definitely felt the inconsistency.

Reply:
I'll try some things and see what feels natural, thanks!

Reply:
Went with `enum class` for both the arguments.